### ISSUE TITLE
"Use of uninitialized value …" warning when running configsmoke.pl

### ISSUE THREAD

**Issue by pjacklam** [abeltje/Test-Smoke#76]
I am running Test-Smoke version 1.80 under Cygwin. I run
```
$ perl /usr/local/bin/configsmoke.pl
```
and when I get to the hostname configuration, I just hit the [ENTER] key to use the default hostname. This gives me two warnings about "Use of uninitialized value":
```
(…)
Use the hostname option to override System::Info->hostname
   Leave empty to use default 'OS-A01-L0123'
[undef] $
Use of uninitialized value $input in pattern match (m//) at /usr/local/bin/configsmoke.pl line 2160, <STDIN> line 12.
Use of uninitialized value $retval in concatenation (.) or string at /usr/local/bin/configsmoke.pl line 2170, <STDIN> line 12.
Got []
(…)
```
The information line right above the prompt says that the default is use the actual hostname (OS-A01-L0123 in my case), but the prompt string says the default is `[undef]`. That seems like a contradiction.

**Comment by abeltje** [comment]
Actually `configsmoke.pl` is no longer maintained, please use the new `tsconfigsmoke.pl`.
With regard to the actual value of 'hostname' in your `smokecurrent_config`, `undef` means use the value from `System::Info`.

**Comment by abeltje** [comment]
This should be resolved in version 1.82
